resources/views/external-link-confirm.blade.php

```diff
@@ -1,8 +1,12 @@
-{{-- External icon classes, dont remove are here for purgecss --}}
-{{-- inline ml-1 -mt-1.5 --}}
+@props([
+    'selector' => null,
```

Comment:
can you detail in the usage file what the selector is for?

Reply:
I think there's 0 docs about external link at this point sadly

Reply:
:sadmici:

Reply:
Done my friend